packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -102,7 +103,7 @@ export function PanelChrome({
   const ariaLabel = title ? selectors.components.Panels.Panel.containerByTitle(title) : 'Panel';
 
   return (
-    <div className={styles.container} style={containerStyles} aria-label={ariaLabel}>
+    <div className={cx(styles.container, dragClass)} style={containerStyles} aria-label={ariaLabel}>
```

Comment:
It seems it would be best to have the dragClass on the header only because otherwise dragging is allowed from the graph as well which also affects the graph: 

https://user-images.githubusercontent.com/13227501/213385223-f222829d-0227-4965-83ab-bb12039b1c7c.mov

Reply:
Good catch 🙌🏾 , I will add the class to the header :)